turftopic/models/decomp.py

```diff
@@ -130,6 +131,85 @@ def fit_transform(
             console.log("Model fitting done.")
         return doc_topic
 
+    def refit_transform(
+        self,
+        n_components: Optional[int] = None,
+        max_iter: Optional[int] = None,
+        random_state: Optional[int] = None,
+    ):
+        """Refits model with the given parameters.
+        This is significantly faster than fitting a new model from scratch.
```

Comment:
Why

Reply:
Huh?

Reply:
I would add documentation on why this is significantly faster